premake5.lua

```diff
@@ -172,6 +172,8 @@
 			{ "ARM64", "ARM64" },
 			{ "x86", "x86 (On macOS, same as x86_64.)" },
 			{ "x86_64", "x86_64" },
+			{ "ppc", "PowerPC 32-bit" },
```

Comment:
This may be better being "ppc32" instead of "ppc", since it makes it sound (in my mind) like 32 bit is the default, when 64 bit is the default on almost all desktop and server class hardware these days.

Reply:
Like with `arm` and `mips`, convention is just `ppc`. Though I won’t really object if there is a strong preference to do otherwise.

Reply:
Good enough reason for me. Let's leave it.